src/server/ntpcontroller.cpp

```diff
@@ -77,12 +96,7 @@ void NtpController::setConnmanProperty(QString key, QString value)
     } else {
         log_debug("Set %s property %s to value %s",
                   QString(CONNMAN_INTERFACE).toStdString().c_str(),
-                  key.toStdString().c_str(),
-                  value.toStdString().c_str());
+                  reply.argumentAt(0).toString().toStdString().c_str(),
+                  reply.argumentAt(1).toString().toStdString().c_str());
     }
```

Comment:
@lpotter: We did libdsme soname fixes lately -> dev env & target need to be in sync. If you have updated both, then it might be: in sbox the post-install ldconfig sometimes fails to run the so-links are not there -> running ldconfig manually should fix that.